exercises/conversions/try_from_into.rs

```diff
@@ -66,71 +63,98 @@ mod tests {
     use super::*;
 
     #[test]
-    #[should_panic]
     fn test_tuple_out_of_range_positive() {
-        let _ = Color::try_from((256, 1000, 10000)).unwrap();
+        assert!(Color::try_from((256, 1000, 10000)).is_err());
     }
     #[test]
-    #[should_panic]
     fn test_tuple_out_of_range_negative() {
-        let _ = Color::try_from((-1, -10, -256)).unwrap();
+        assert!(Color::try_from((-1, -10, -256)).is_err());
+    }
+    #[test]
+    fn test_tuple_sum() {
+        assert!(Color::try_from((-1, 255, 255)).is_err());
     }
     #[test]
     fn test_tuple_correct() {
-        let c: Color = (183, 65, 14).try_into().unwrap();
+        let c: Color = match (183, 65, 14).try_into() {
+            Ok(color) => color,
+            Err(_) => Color {
+                red: 0,
+                green: 0,
+                blue: 0,
+            },
+        };
         assert_eq!(c.red, 183);
         assert_eq!(c.green, 65);
         assert_eq!(c.blue, 14);
     }
-
     #[test]
-    #[should_panic]
     fn test_array_out_of_range_positive() {
-        let _: Color = [1000, 10000, 256].try_into().unwrap();
+        let c: Color = [1000, 10000, 256].try_into();
+        assert!(c.is_err());
     }
     #[test]
-    #[should_panic]
     fn test_array_out_of_range_negative() {
-        let _: Color = [-10, -256, -1].try_into().unwrap();
+        let c: Color = [-10, -256, -1].try_into();
+        assert!(c.is_err());
+    }
+    #[test]
+    fn test_array_sum() {
+        let c: Color = [-1, 255, 255].try_into();
+        assert!(c.is_err());
     }
     #[test]
     fn test_array_correct() {
-        let c: Color = [183, 65, 14].try_into().unwrap();
+        let c: Color = match [183, 65, 14].try_into() {
+            Ok(color) => color,
+            Err(_) => Color {
+                red: 0,
+                green: 0,
+                blue: 0,
+            },
+        };
```

Comment:
I know bikesheding is tiresome but would definitely prefers:

[Playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=%23%5Bderive(Debug%2C%20PartialEq)%5D%0Astruct%20Color%20%7B%0A%20%20%20%20red%3A%20u8%2C%0A%20%20%20%20green%3A%20u8%2C%0A%20%20%20%20blue%3A%20u8%2C%0A%7D%0A%0A%23%5Bcfg(test)%5D%0Amod%20tests%20%7B%0A%20%20%20%20use%20super%3A%3A*%3B%0A%20%20%20%20%23%5Btest%5D%0A%20%20%20%20fn%20test_coulour()%20%7B%0A%20%20%20%20%20%20%20%20let%20c%20%3A%20std%3A%3Aresult%3A%3AResult%3CColor%2C%20String%3E%20%3D%20Ok(Color%20%7Bred%3A%20182%2C%20green%3A%2065%2C%20blue%3A%2014%7D)%3B%0A%20%20%20%20%20%20%20%20assert_eq!(c%2C%20Ok(Color%20%7B%20red%20%3A%20183%2C%20green%3A%2065%2C%20blue%3A%2014%20%7D))%3B%0A%0A%20%20%20%20%2F%2F%20or%0A%20%20%20%20%7D%0A%7D%0A)
```rust
let c : std::result::Result<Color, String> = Ok(Color {red: 182, green: 65, blue: 14});
assert_eq!(c, Ok(Color { red : 183, green: 65, blue: 14 }));
```
Failing message will be better than zeroish value.
Rustlings is used as a basis for new rustacean, it's important to try to stick with nice and idiomatic way of using rust :D

Reply:
Agreed. Changed in the latest commit :heart: